### DISCUSSION TITLE
Chapter 3 Ingress: specified service port

### DISCUSSION THREAD

**Discussion by dchebakov** [digitalocean/Kubernetes-Starter-Kit-Developers#149] [General]
Thank you for the excellent tutorial!

Could you please explain why we specify Ingress host listening port [8080](https://github.com/digitalocean/Kubernetes-Starter-Kit-Developers/blob/main/03-setup-ingress-controller/assets/manifests/nginx/echo_host.yaml#L23), but actually our service listening on port [80](https://github.com/digitalocean/Kubernetes-Starter-Kit-Developers/blob/f9b90249657b7df86a100ade530095a42052b265/03-setup-ingress-controller/assets/manifests/nginx/echo_service.yaml#L10)? How does it work when our service is accessible for the Ingress on port 8080 when it is listening on port 80?

**Comment by mtiutiu-heits** [comment]
Hi @dchebakov 

Can be a mistake from our end. Let me take a look and get back to you.

If I understood your question correctly, my answer is as follows: yes, the Ingress Controller must be connected with ports exposed by Kubernetes services.

Thanks.

**Comment by v-ctiutiu** [comment]
@dchebakov 

Seems to be a mistake on our end. Thanks for highlighting the issue. I opened PR #152 to address the issue.
